src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -366,6 +366,19 @@ private DecisionDagBuilder(CSharpCompilation compilation, LabelSymbol defaultLab
             }
         }
 
+        private void MakeTestsAndBindings(
+            BoundDagTemp input,
+            BoundTypePattern typePattern,
+            ArrayBuilder<BoundDagTest> tests,
+            ArrayBuilder<BoundPatternBinding> bindings)
+        {
+            TypeSymbol type = typePattern.DeclaredType.Type;
+            SyntaxNode syntax = typePattern.Syntax;
+
+            // Add a null and type test
+            input = MakeConvertToType(input, syntax, type, tests);
```

Comment:
This assignment doesn't look used

Reply:
Agreed, it could be discarded in the code as it stands.  However in a PR under development we'll use it, as it becomes the input to a subsequent pattern after an `and` operator.

---
In reply to: [360575780](https://github.com/dotnet/roslyn/pull/40503#discussion_r360575780) [](ancestors = 360575780)